node/src/chain_spec.rs

```diff
@@ -23,6 +23,11 @@ use serde::{Deserialize, Serialize};
 use sp_core::{crypto::UncheckedInto, sr25519, Pair, Public};
 use sp_runtime::traits::{IdentifyAccount, Verify};
 
+/// The ID of this Parachain as registered on the Relay Chain.
+///
+/// TODO: When we register as a common good parachain this will need to change.
+pub const PARA_ID: u32 = 12648430;
```

Comment:
I think there is already a parachain id which @lovelaced assigned for the Rococo deployment?

Reply:
Canvas on Rococo is 1002 - and as far as I understand common good chains should have a para ID 1000 <= x < 2000.